### DISCUSSION TITLE
Frame selection Howto

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#356] [Ideas]
Hi, 
I think I have a solution for selecting frames that can be usefull . I recorded a Session to show how it works.
The only thing that would it make better, is to have the Birthday and Wedding Button on the main Screen.
And the browser refresh ?

What do you think ?

 

https://github.com/user-attachments/assets/cbcaafa3-682b-4cdc-8f66-3d3ed9c7fdaa

**Comment by Striker62** [comment]
If the frame is cached it will not change in the also if you refresh the window.
To avoid this add ?v=1.0 to the definition .... frame.png?v=1.0

**Comment by McKaiver** [comment]
looks good, but i can't find the private.css in the folder, so i created one by myself:
```
/*zoom livestream */
/*
##preview-stream {
	transform: scale(1.45)
}
*/
/*#q-app{
	transform: scale(1)
}
*/
.action-button {
	font-size: 1.0rem;
}
.action-button *>.q-icon {
	font-size: 4.5rem
}

#preview-stream::after {
	content: "";
	position: absolute;
	top: 0;
	left: 0;
	right: 0;
	bottom: 0;
	background-image: url('/userdata/frames/freigestellt_transparent.png'); /* Dein PNG als Overlay */
	background-size: cover; /* Deckt den gesamten Bereich ab */
	background-position: center;
	pointer-events: none; /* Erlaubt weiterhin die Interaktion mit dem Stream */
	z-index: 1; /* Stellt sicher, dass das Overlay über dem Stream liegt */
	opacity: 0.9; /* Optional: Transparenz für das Overlay */
/*	transform: scaleX(-1); /* Horizontal spiegeln */
```
![grafik](https://github.com/user-attachments/assets/6c0b794c-39a9-45c6-a178-57fb519db477)

![grafik](https://github.com/user-attachments/assets/f77aaf75-5fcd-45db-ac5a-82dc0b5ab6d9)

![grafik](https://github.com/user-attachments/assets/cdac3c33-22d6-4e26-853b-c4044b6097a0)

**Comment by Striker62** [comment]
You can find the private.css here
![Photobooth-App – Mozilla Firefox 01 10 2024 11_17_11](https://github.com/user-attachments/assets/6341a13c-ac05-4bed-bdc8-6192b6b13fcb)

**Reply by McKaiver** [reply]
Yes, i had no file there, so i made it manually. I guess i will do a new installation and try again, maybe i've messed up something

**Reply by McKaiver** [reply]
ok, i made a complete new installation, new immage and so on.

here the commands:
`sudo apt update
sudo apt upgrade
sudo usermod --append --groups tty,input $(whoami)
sudo apt -y install ffmpeg libturbojpeg0 python3-pip libgl1 libgphoto2-dev fonts-noto-color-emoji rclone inotify-tools
sudo apt -y install pipx

pipx ensurepath

Evtl Neustart nötig

pipx install --system-site-packages photobooth-app --pip-args='--prefer-binary'


Erster Start

mkdir ~/photobooth-data
cd ~/photobooth-data
photobooth`

Now it looks more like your menus, because previously i had different menu items:
![grafik](https://github.com/user-attachments/assets/dcc4000e-131b-4dab-98f4-b647206f3cd7)

But still no private.css

![grafik](https://github.com/user-attachments/assets/38bf6060-4b5d-40e4-b4c3-7303a95c8f91)


is it, because I'm using not the user Pi, I created the user Fotobox

**Reply by mgineer85** [reply]
You should be able to browse to the css: /private.css in the browser and see a placeholder or, when exists see the content.
https://photobooth-app.org/reference/customizetheme/

**Reply by McKaiver** [reply]
ok, now it works...

i manually placed the file there. Last time i did that, i wrote the settings manually in it and it seems, that there was a Typo or somethin, now i copied it and it works.
